### ISSUE TITLE
Adapting dataset from Conv2D tutorial to (n, 25, 32, 4)

### ISSUE THREAD

**Issue by vladimirdlc** [keras-team/keras#8153]
I'm trying to adapt the Conv2D tutorial dataset from the MNIST (60000, 28, 28, 1) to another dataset that has the shaped of (14422, 25, 32, 4) but using the same filters I cannot parse the data. I wanna try with a small filter for start and test, but this small one doesn't even seem to work at all... is this datasate shape possible to convolve in Keras?

Any help would be appreciated,
Here is the tutorial code as https://blog.keras.io/building-autoencoders-in-keras.html

input_img = Input(shape=(28, 28, 1))  # adapt this if using `channels_first` image data format
x = Conv2D(32, (3, 3), activation='relu', padding='same')(input_img)
x = MaxPooling2D((2, 2), padding='same')(x)
x = Conv2D(32, (3, 3), activation='relu', padding='same')(x)
encoded = MaxPooling2D((2, 2), padding='same')(x)
# at this point the representation is (7, 7, 32)
x = Conv2D(32, (3, 3), activation='relu', padding='same')(encoded)
x = UpSampling2D((2, 2))(x)
x = Conv2D(32, (3, 3), activation='relu', padding='same')(x)
x = UpSampling2D((2, 2))(x)
decoded = Conv2D(1, (3, 3), activation='sigmoid', padding='same')(x)

autoencoder = Model(input_img, decoded)
autoencoder.compile(optimizer='adadelta', loss='binary_crossentropy')

----modded to
input_frame = Input(shape=trainingData[0].shape)
#input_img = Input(shape=(28, 28, 1))  # adapt this if using `channels_first` image data format
x = Conv2D(32, (3, 3), activation='tanh', padding='same')(input_frame)
x = MaxPooling2D((2, 2), padding='same')(x)
x = Conv2D(32, (3, 3), activation='tanh', padding='same')(x)
encoded = MaxPooling2D((2, 2), padding='same')(x)
x = Conv2D(32, (3, 3), activation='tanh', padding='same')(encoded)
x = UpSampling2D((2, 2))(x)
x = Conv2D(32, (3, 3), activation='tanh', padding='same')(x)
x = UpSampling2D((2, 2))(x)
decoded = Conv2D(4, (3, 3), activation='tanh', padding='same')(x)
autoencoder = Model(input_frame, decoded)
autoencoder.compile(optimizer='rmsprop', loss='mse', metrics=['accuracy'])
